src/ucp/stream/stream_recv.c

```diff
@@ -277,34 +277,17 @@ ucp_stream_am_rdesc_get(ucp_worker_t *worker, void *data, size_t length,
     return rdesc;
 }
 
-static UCS_F_ALWAYS_INLINE ucs_status_ptr_t
-ucp_stream_am_get_ep(ucp_worker_t *worker, uint64_t sender_uuid,
-                     ucp_recv_desc_t *rdesc)
+static UCS_F_ALWAYS_INLINE ucp_ep_h
+ucp_stream_am_get_ep(ucp_worker_t *worker, uint64_t sender_uuid)
```

Comment:
just found that we already have such a func: [ucp_worker_ep_find](https://github.com/openucx/ucx/blob/2908fbdef866db4b3a3e79f899365ae2af1cf412/src/ucp/core/ucp_worker.h#L204)